src/site.css

```diff
@@ -128,10 +128,6 @@ nav.active {
   @apply inline-block mr-2 text-text fill-primary;
 }
 
-#readme .hljs {
-  @apply text-text bg-secondary;
```

Comment:
Does this CSS get pre-processed? I wasn't aware that `@apply` was being used anywhere — it was [proposed but withdrawn](https://www.xanthir.com/b4o00).

Reply:
Ah, it's Tailwind. (*\*shudder\**) Never mind!

Reply:
Yes this CSS file is processed before being used in production (or dev for that matter).

We use `tailwind` to process our CSS file, and `@apply` is a directive within `tailwind`

You can look in our `package.json` to see the ` "postInstall": "npm run tailwind:build && npm run mini.js && npm run mini:css"`

Those above steps use `tailwind` to process the CSS file and output it to `./public/site.css` then the other two steps both use `minify` to minify our production JavaScript and CSS into `./public/site.min.js` and `./public/site.min.css` respectively